### ISSUE TITLE
Rename "Timer" to "Stopwatch"

### ISSUE THREAD

**Issue by samreid** [phetsims/wave-on-a-string#137]
From https://github.com/phetsims/scenery-phet/issues/530, we standardized on the time measurement tool name as "Stopwatch".  This sim still refers to it as "Timer" in the checkbox and maybe in some variable names.  It would be good to use the new convention.  Assigned to @ariel-phet for priority and delegation.

**Comment by samreid** [comment]
In the aforementioned issue, @arouinfar said:

> @samreid while perhaps a bit verbose, StopwatchNode would be more accurate. Generally, a timer counts down to zero, while a stopwatch counts up.

and 

> I would also be in favor of changing the string in WOAS to "Stopwatch", though it's not particularly urgent.

**Comment by ariel-phet** [comment]
It seems this should be done as part of future work on WOAS (such as instrumentation) - marking deferred

**Comment by jonathanolson** [comment]
Implemented. I renamed the string in master, but I can also create a new stopwatch string (or reuse a common one) if we want to force translations to update. Thoughts?

**Comment by ariel-phet** [comment]
@jonathanolson if you change in master, when the sim is republished won't translators need to update anyhow? Regardless I do not think we need to "force" it, timer and stopwatch are pretty close in meaning.

**Comment by jonathanolson** [comment]
> if you change in master, when the sim is republished won't translators need to update anyhow?

If I just change the English string (keeping the string key), when published it will update the English translation and will use the old translations.

If I change things to a new string key, when published it will use the English translation for everything (and will become un-translated for other translations, which would need to be updated).

> Regardless I do not think we need to "force" it, timer and stopwatch are pretty close in meaning.

Sounds good, I kept the same string key so I believe that should be the desired behavior. Closing.
